### PR TITLE
Adds Docker Development Scripts

### PR DESCRIPTION
**Pull request by StetsenTech**
**Ticket(s)**:
https://app.asana.com/0/0/1147512938357231/f

**Summary**:
- Adds Docker Scripts to help with development
  - Pretty much the same as in other environments

### CODE REVIEW

**Comment by rshipp** [comment]
do we need these for setup.sh? i thought we had talked about not using them anymore

File: docker-compose/scripts/dbconnect.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# This connects to the development database. The user and database name are in docker-compose.yml
+docker-compose exec platformdb psql -U user password
```
Comment (sadleb) [inline docker-compose/scripts/dbconnect.sh]: Does this actually work? I thought the command to send a password was -p<insert password>

**Comment by StetsenTech** [comment]
You don't need them. Just nice to haves if you don't want to manually run commands. Honestly, I don't remember a conversation about not using them anymore. I made the PR so that this is inline with our current development environments.

File: docker-compose/scripts/dbconnect.sh (hunk 1 above)
Comment (StetsenTech) [reply docker-compose/scripts/dbconnect.sh]: You are right. That shouldn't be there. Works if it isn't there.

**Pull request merged**